teste2e/main_test.go

```diff
@@ -0,0 +1,41 @@
+// Go Substrate RPC Client (GSRPC) provides APIs and types around Polkadot and any Substrate-based chain RPC calls
+// Copyright (C) 2019  Centrifuge GmbH
+//
+// This file is part of Go Substrate RPC Client (GSRPC).
+//
+// GSRPC is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// GSRPC is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+package teste2e
+
+import (
+	"fmt"
+	"testing"
+
+	gsrpc "github.com/centrifuge/go-substrate-rpc-client"
+	"github.com/centrifuge/go-substrate-rpc-client/config"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetBlockHashAndVersion(t *testing.T) {
```

Comment:
are we not testing the metadata call?

Reply:
does it matter for this specific one?

Reply:
I thought its part of the deliverable?

Reply:
Ok ignore this, I see it's calling `state.getVersion` which tests metadata :) 
sorry for the confusion.